doc/rbd/iscsi-requirements.rst

```diff
@@ -0,0 +1,53 @@
+==========================
+iSCSI Gateway Requirements
+==========================
+
+To implement the Ceph iSCSI gateway there are a few requirements. Red
+Hat recommends using two nodes for a highly available Ceph iSCSI gateway
+solution and up to four Ceph iSCSI gateway nodes.
+
+For hardware recommendations, see the `Hardware Recommendation page <http://docs.ceph.com/docs/master/start/hardware-recommendations/>`_
+for more details.
+
+.. WARNING::
+    On the iSCSI gateway nodes, the memory footprint of the RBD images
+    can grow to a large size. Plan memory requirements accordingly based
+    off the number RBD images mapped. Each RBD image roughly uses 90 MB
```

Comment:
@dillaman : Sounds good.  I'll wait to hear back from you or Mike on this one, before I update the source content.